src/router/index.js

```diff
@@ -1,10 +1,26 @@
-import './App.css';
+import "./index.css";
+import {BrowserRouter as Router, Switch, Route} from "react-router-dom";
+import LoginPage from "./pages/LoginPage";
+import PostsPage from "./pages/PostsPage";
+import PostDetailsPage from "./pages/PostDetailsPage";
+import ProfilePage from "./pages/ProfilePage";
 
 function App() {
   return (
-    <div className="App">
-     {/* router goes here */}
-    </div>
+    <Router>
+      <div className="App">
+        <Switch>
+          <Route exact path="/" component={LoginPage}></Route>
+          <Route exact path="/postsPage" component={PostsPage}></Route>
+          <Route
+            exact
+            path="/postDetailsPage"
+            component={PostDetailsPage}
+          ></Route>
+          <Route exact path="/profilePage" component={ProfilePage}></Route>
```

Comment:
You can close tags on itself rather then a separate tag. that is one of the benefit of using JSX.

`<Route exact path="/" component={LoginPage} />`

Reply:
changes made,  removed all  route closing tags